test/README.md

```diff
@@ -189,6 +197,22 @@ go test -v -tags=e2e -count=1 ./test/conformance --dockerrepo gcr.myhappyproject
 go test -v -tags=e2e -count=1 ./test/e2e --dockerrepo gcr.myhappyproject
 ```
 
+### Using a docker tag
+
+The default docker tag used for the test images is `latest`, which can
+be problematic on [Minikube]. To avoid having to configure a remote
+container registry to support the `Always` pull policy for `latest`
+tags, you can have the tests use a specific tag:
+
+```bash
+go test -v -tags=e2e -count=1 ./test/conformance --tag any-old-tag
+go test -v -tags=e2e -count=1 ./test/e2e --tag any-old-tag
+```
+
+Of course, this implies that you tagged the images when you [uploaded
+them](#building-the-test-images):
```

Comment:
nit: either the ":" should be "." or there are some instruction missing after it?

Reply:
Good catch, fallout from previous edits where I had a sample command following it. Will fix